GDAXSharp/Services/Withdrawals/Models/Coinbase.cs

```diff
@@ -1,11 +1,16 @@
-﻿namespace GDAXSharp.Services.Withdrawals.Models
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace GDAXSharp.Services.Withdrawals.Models
 {
     public class Coinbase
     {
-        public decimal amount { get; set; }
+        public decimal Amount { get; set; }
 
-        public string currency { get; set; }
+        [JsonConverter(typeof(StringEnumConverter))]
+        public Shared.Currency Currency { get; set; }
```

Comment:
need to add `[JsonProperty("currency")]` here otherwise it won't serialize properly